README.md

```diff
@@ -1914,6 +1916,17 @@ Simple text message
  }
 ```
 
+```json
+ {
+    "message": "Message with image downloaded from URL: digest authentication",
+    "imageurl": "<image url>",
+    "auth": "digest",
+    "user": "myspecialuser",
+    "password": "myspecialpassword"
+ }
+```
+For the above example, I would only recommend this be used in a local MQTT server instance, as your password is being transmissed for your imaging device.
```

Comment:
`transmissed` is a typo. ;)

I think you'll want to add "transmitted in the clear" so that people understand.

Reply:
And this I've made clearer also. 

Again sorry, I don't know how to add the relevant section